Spring: incompability with 2.2.6.RELEASE

If I try to update sample-maven-spring with spring-boot-starter-web 2.2.6.RELEASE instead of provided 2.1.1.RELEASE, build fails with following error:

```
[INFO] --- maven-compiler-plugin:3.7.0:compile (default-compile) @ sample-maven-spring ---
[INFO] Changes detected - recompiling the module!
[WARNING] File encoding has not been set, using platform encoding Cp1250, i.e. build is platform dependent!
[INFO] Compiling 1 source file to C:\Users\Kamil\Desktop\tsge\sample-maven-spring\target\classes
[INFO]
[INFO] --- typescript-generator-maven-plugin:2.21.588:generate (generate) @ sample-maven-spring ---
Running TypeScriptGenerator version 2.21.588
Loading class cz.habarta.typescript.generator.ext.AxiosClientExtension
Configuration: 'AxiosClientExtension' extension set 'generateJaxrsApplicationClient' parameter to 'true'
Configuration: 'AxiosClientExtension' extension set 'restResponseType' parameter to 'Promise<Axios.GenericAxiosResponse<R>>'
Configuration: 'AxiosClientExtension' extension set 'restOptionsType' parameter to '<O>'
Scanning Spring application: cz.habarta.typescript.generator.sample.spring.SpringTestApplication
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 4.595 s
[INFO] Finished at: 2020-03-30T18:01:58+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal cz.habarta.typescript-generator:typescript-generator-maven-plugin:2.21.588:generate (generate) on project sample-maven-spring: Execution generate of goal cz.habarta.typescript-generator:typescript-generator-maven-plugin:2.21.588:generate failed: Cannot instantiate interface org.springframework.context.ApplicationListener : org.springframework.boot.cloud.CloudFoundryVcapEnvironmentPostProcessor: class org.springframework.boot.cloud.CloudFoundryVcapEnvironmentPostProcessor is not assignable to interface org.springframework.context.ApplicationListener -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

I haven't investigated it further yet.

--- a/sample-maven-spring/pom.xml
+++ b/sample-maven-spring/pom.xml
@@ -9,14 +9,14 @@
     <name>sample-maven-spring</name>
 
     <properties>
-        <typescript-generator.version>2.12-SNAPSHOT</typescript-generator.version>
+        <typescript-generator.version>2.21-SNAPSHOT</typescript-generator.version>
     </properties>
 
     <dependencies>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
-            <version>2.1.1.RELEASE</version>
+            <version>2.2.6.RELEASE</version>
         </dependency>
     </dependencies>
 

--- a/typescript-generator-core/pom.xml
+++ b/typescript-generator-core/pom.xml
@@ -16,7 +16,7 @@
         <javax.activation.version>1.2.0</javax.activation.version>
         <jaxb.version>2.3.1</jaxb.version>
         <jackson1.version>1.9.13</jackson1.version>
-        <jackson2.version>2.9.7</jackson2.version>
+        <jackson2.version>2.10.3</jackson2.version>
         <jersey.version>2.27</jersey.version>
     </properties>
 

--- a/typescript-generator-spring/pom.xml
+++ b/typescript-generator-spring/pom.xml
@@ -20,7 +20,7 @@
             <dependency>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-dependencies</artifactId>
-                <version>2.1.9.RELEASE</version>
+                <version>2.2.6.RELEASE</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>

